src/fillTank.test.js

```diff
@@ -1,11 +1,48 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+let customer;
 
+  beforeEach(() => {
+    customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
   });
 
-  // write tests here
+  it('should fill tank fully if no amount provided', () => {
+    fillTank(customer, 50);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+    expect(customer.money).toBe(1400);
```

Comment:
The test case 'should fill tank fully if no amount provided' does not seem to check if no amount is provided. You are passing 50 as the second argument to `fillTank`. You should remove this argument to accurately test the scenario described.